cafebabel/tests/test_drafts.py

```diff
@@ -123,18 +120,16 @@ def test_draft_editing_should_update_content(client, user, editor):
         'title': 'My article',
         'summary': 'Summary',
         'body': 'Article body',
-        'language': 'en',
         'author': user.id
     }
-    draft = Article.objects.create(**data)
+    draft = Article.objects.create(**data, language='en')
     updated_data = data.copy()
-    updated_data['language'] = 'fr'
     response = client.post(f'/en/article/draft/{draft.id}/edit/',
                            data=updated_data, follow_redirects=True)
     assert response.status_code == 200
     updated_draft = Article.objects.get(id=draft.id)
     assert updated_draft.id == draft.id
-    assert updated_draft.language == 'fr'
+    assert updated_draft.language == 'en'
```

Comment:
I don't think that assertion is still pertinent.

Reply:
this whole test is useless as long as we don't actually modify the data that we pass…